src/app/services/progressesApi.ts

```diff
@@ -0,0 +1,19 @@
+import { api } from './api';
+
+export const progressesApi = api.injectEndpoints({
```

Comment:
In line headers: { credentials: 'include' }. This line is setting the credentials header to include, which tells the browser to include the credentials from the current request when making the POST request to /progresses. However, the credentials header is only necessary when making a cross-origin request. Since the /progresses endpoint is on the same origin as the current request, there is no need to set the credentials header. Therefore, you can remove the headers: { credentials: 'include' }

Reply:
The cookie we set is a HTTP Only cookie, accessing it directly in frontend and setting it in headers is not possible thus credentials include will do the work for us